server/spinmint.go

```diff
@@ -173,6 +173,61 @@ func waitForBuildAndSetupSpinmint(pr *model.PullRequest) {
 	commentOnIssue(pr.RepoOwner, pr.RepoName, pr.Number, message)
 }
 
+func waitForBuildAndSetupSpinmintUpgrade(pr *model.PullRequest) {
```

Comment:
Can we reuse this code instead of copy paste and just parameterize this function?

Reply:
yep, will change

Reply:
done